framework/src/modules/dpos_v2/constants.ts

```diff
@@ -26,3 +26,10 @@ export const COMMAND_ID_VOTE = 1;
 export const COMMAND_ID_UNLOCK = 2;
 export const COMMAND_ID_POM = 3;
 export const COMMAND_ID_UPDATE_GENERATOR_KEY = 4;
+
+export const WAIT_TIME_VOTE = 2000;
+export const WAIT_TIME_SELF_VOTE = 260000;
+export const VOTER_PUNISH_TIME = 260000;
+export const SELF_VOTE_PUNISH_TIME = 78000;
```

Comment:
This should be `780000` I think, one 0 is missing at the end.

Reply:
Good catch!